content/language-agnostic/encyclopedia/what-is-a-parameter-and-an-argument?.md

```diff
@@ -0,0 +1,28 @@
+<!--
+---
+Subjects:
+  - "What is a parameter and an argument?"
+Tags: 
+  - "language-agnostic"
+  - "functions"
+Catalog Content:  # If listing multiple URLs, please put the most relevant one first 
+  - "https://www.codecademy.com/courses/introduction-to-javascript/lessons/functions/exercises/parameters"
```

Comment:
Please stick to just course and path landing pages
From `documentation.md`: "URL(s) of Codecademy course and path landing pages that relate to the entry.
- Please avoid linking to individual content items, because their URLs may change and some are gated by Pro membership"